sources/packages/backend/apps/api/src/route-controllers/designation-agreement/designation-agreement.controller.service.ts

```diff
@@ -59,19 +59,19 @@ export class DesignationAgreementControllerService {
           designationLocationId: agreementLocation.id,
         }),
       ),
-    } as GetDesignationAgreementDto;
+    } as DesignationAgreementAPIOutDTO;
```

Comment:
We try to avoid the cast in the DTO return to avoid sending unexpected default values.